packaging/docker/samples/mounted-volume/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3'
+services:
```

Comment:
Given the topic of this PR/issue, I had expected to see something about configuring mounted volumes in this docker-compose file, which I don't see, or do I just not understand docker?